src/utils/password.expiration.cron.job.ts

```diff
@@ -60,26 +46,22 @@ export const checkPasswordExpiration = async () => {
 
     }
 
-    eventEmitter.emit('passwordExpirationCheckCompleted', users.length);
   } catch (error) {
     console.error('Error checking password expiration:', error);
   }
 };
 
-const startCronJob = () => {
-  if (process.env.NODE_ENV === 'development') {
-    cron.schedule(
-      '0 0 * * *',
-      async () => {
-        console.log('Running password expiry cron job at 00:00 at Africa/Kigali timezone');
-        await checkPasswordExpiration();
-      },
-      {
-        scheduled: true,
-        timezone: 'Africa/Kigali',
-      }
-    );
-  }
-};
 
-export default startCronJob;
\ No newline at end of file
+if (process.env.NODE_ENV === 'development') {
```

Comment:
Does this mean we will only check for the password expiration in development?

Reply:
This change were made by Olivier, let ask him, but from what I see the condition check true only on development

Reply:
But then again will we check for the password expiration in production if it's commented? @Oliviier-dev

Reply:
Yes, it would check for password expiration in all environments if we commented the if condition; the only problem would be a failure of the CI. @SergeInGithub

Reply:
@Oliviier-dev The password expiration feature was assigned to you... this is a shortcut you're doing and it would mean that you task wouldn't have been done!! Find a way to fix this for both environments.